tests/validation/config.json

```diff
@@ -1222,57 +1296,75 @@
             "title": "Instances + motion blur",
             "renderCount": 10,
             "playgroundId": "#YB006J#404",
-            "excludedEngines": ["webgl1"],
+            "excludedEngines": [
+                "webgl1"
+            ],
             "referenceImage": "instances-mb.png"
         },
         {
             "title": "Instances + GBR + motion blur",
             "renderCount": 10,
             "playgroundId": "#YB006J#403",
-            "excludedEngines": ["webgl1"],
+            "excludedEngines": [
+                "webgl1"
+            ],
             "referenceImage": "instances-gbr-mb.png"
         },
         {
             "title": "Instances manual update + motion blur",
             "renderCount": 10,
             "playgroundId": "#HJGC2G#50",
-            "excludedEngines": ["webgl1", "webgpu"],
+            "excludedEngines": [
+                "webgl1",
+                "webgpu"
+            ],
             "referenceImage": "instances-mb-manual.png"
         },
         {
             "title": "Thin instances + motion blur",
             "renderCount": 30,
             "playgroundId": "#HJGC2G#60",
-            "excludedEngines": ["webgl1", "webgpu"],
+            "excludedEngines": [
+                "webgl1",
+                "webgpu"
+            ],
             "referenceImage": "thin-instances-mb.png"
         },
         {
             "title": "Thin instances + motion blur + manual",
             "renderCount": 30,
             "playgroundId": "#HJGC2G#61",
-            "excludedEngines": ["webgl1"],
+            "excludedEngines": [
+                "webgl1"
+            ],
             "referenceImage": "thin-instances-mb-manual.png"
         },
         {
             "title": "Thin instances + dynamic buffer resize",
             "renderCount": 10,
             "playgroundId": "#217750#33",
-            "excludedEngines": ["webgl1"],
+            "excludedEngines": [
+                "webgl1"
+            ],
             "referenceImage": "thin-instances-buffer-resize.png"
         },
         {
             "title": "Instances + render self motion blur",
             "renderCount": 10,
             "playgroundId": "#217750#34",
-            "excludedEngines": ["webgl1"],
+            "excludedEngines": [
+                "webgl1"
+            ],
             "referenceImage": "instances-renderself-mb.png"
         },
         {
             "title": "Thin instances + render self motion blur",
             "renderCount": 10,
             "playgroundId": "#217750#35",
-            "excludedEngines": ["webgl1"],
+            "excludedEngines": [

```

Comment:
hmm, looks like the autoformatter is expanding the arrays, however I'm not super oppsed to this change in format...

Reply:
Ehh, nevermind.